README.rst

```diff
@@ -37,7 +40,15 @@ Uninstall
 - **Whichever the install mode**::
 
     python3 -m pip uninstall psydac
+
+Running tests
+*********
+- **Setup**::
     
+   export PSYDAC_MESH_DIR=/path/to/psydac/mesh/
+   python -m pytest --pyargs psydac -m "not parallel"
+   python /path/to/psydac/mpi_tester.py --pyargs psydac -m "parallel"
```

Comment:
We should probably change `python` with `python3`

Reply:
I didn't what made more sense because I assume that most people only have python 3 and use python as a way to launch the python3 executable.